api/v1beta1/hetznerbaremetalhost_types.go

```diff
@@ -230,8 +232,9 @@ type HetznerBareMetalHostSpec struct {
 	// +optional
 	ConsumerRef *corev1.ObjectReference `json:"consumerRef,omitempty"`
 
-	// MaintenanceMode indicates that a machine is supposed to be deprovisioned
-	// and won't be selected by any Hetzner bare metal machine.
+	// MaintenanceMode indicates that a machine is supposed to be deprovisioned. The CAPI Machine
+	// will get the cluster.x-k8s.io/remediate-machine annotation, and CAPI will deprovision the
+	// machine. Accordingly, the host won't be selected by any Hetzner bare metal machine.
```

Comment:
Accordingly -> Additionally

I am not sure what you mean by accordingly, but for me "additionally" is correct. Do you agree?

I think setting the remediate condition on the Machine object is unrelated to the host not being chosen anymore